stdlib/tkinter/__init__.pyi

```diff
@@ -3064,11 +3064,33 @@ class Text(Widget, XView, YView):
     def edit_separator(self) -> None: ...  # actually returns empty string
     def edit_undo(self) -> None: ...  # actually returns empty string
     def get(self, index1: _TextIndex, index2: _TextIndex | None = None) -> str: ...
-    # TODO: image_* methods
-    def image_cget(self, index, option): ...
-    def image_configure(self, index, cnf: Incomplete | None = None, **kw): ...
-    def image_create(self, index, cnf={}, **kw): ...
-    def image_names(self): ...
+    def image_cget(self, index: _TextIndex, option: str) -> str | int: ...  # image returned as string, others return str, int.
```

Comment:
This is annoying to use, because you need something like:

```
align = text.image_cget(image_name, 'align')
assert isinstance(align, str)
# do something with align
```

Basically, when a user gets a type `str | int`, type checkers do not let the user just assume it is a string. They think you need to somehow handle both cases.

It would be better to do:

```suggestion
    def image_cget(self, index: _TextIndex, option: str) -> Any: ...  # image returned as string, others return str, int.
```

Or maybe:

```suggestion
    @overload
    def image_cget(self, index: _TextIndex, option: Literal["align", "image", "name"]) -> str: ...
    @overload
    def image_cget(self, index: _TextIndex, option: Literal["padx", "pady"]) -> int: ...
    @overload
    def image_cget(self, index: _TextIndex, option: str) -> Any: ...
```